### PR TITLE
feat: blog post on query language

### PR DESCRIPTION
**Pull request by TheCatLady**
# Description

Blog post by @aquamatthias ✨

# To-Dos

- [x] Lint with `yarn lint`
- [x] Build successfully with `yarn build`

# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://resoto.com/code-of-conduct).

### CODE REVIEW

**Review by aquamatthias — APPROVED** [review]
Thanks @TheCatLady for all the work!

**Comment by scapecast** [comment]
suggesting to change the initial paragraph. In general, we should always use the proper AWS product names, and show value by addition. 

> retrieving information about resources you have deployed in your [AWS](https://aws.amazon.com) infrastructure means tediously navigating the [AWS](https://aws.amazon.com) web console or using their CLI.

> retrieving information about resources you have deployed in your [AWS](https://aws.amazon.com) infrastructure means navigating the AWS Management Console or using the AWS Command Line Interface. That approach works well in a single account set-up. However, the recommended best practice for AWS is to set up a multi-account environment - and as the number of accounts grows, navigating your infrastructure and finding resources via the Console or the CLI becomes hard to impossible.

File: blog/2022-02-04-resoto-query-language-101/index.md [hunk 1]
```diff
@@ -13,7 +12,7 @@ import Image from '@theme/IdealImage';
 
 <Image img={require('./img/banner.png')} alt="Left: Sheep Spinning Up Cloud Resources; Right: Confused Sheep with Abacus" />
 
-Retrieving information about resources you have deployed in your [AWS](https://aws.amazon.com) infrastructure means tediously navigating the [AWS](https://aws.amazon.com) web console or using their CLI. Every product has its way of navigation and data to show. The process becomes cumbersome quickly, especially if you have infrastructure in more than one region or account since you need to switch explicitly for every combination.
+Retrieving information about resources you have deployed in your [AWS](https://aws.amazon.com) infrastructure means tediously navigating the [AWS](https://aws.amazon.com) Management Console or using the AWS Command Line Interface. That approach works well in a single account setup. However, the recommended best practice for [AWS](https://aws.amazon.com) is to set up a multi-account environment - and as the number of accounts grows, navigating your infrastructure and finding resources via the Console or the CLI becomes hard to impossible.
```
Comment (TheCatLady) [inline blog/2022-02-04-resoto-query-language-101/index.md:15]:
```suggestion
Retrieving information about resources you have deployed in your [AWS](https://aws.amazon.com) infrastructure means tediously navigating the [AWS Management Console](https://aws.amazon.com/console) or using the [AWS Command Line Interface](https://aws.amazon.com/cli). That approach works well in a single account setup. However, the recommended best practice for [AWS](https://aws.amazon.com) is to set up a multi-account environment - and as the number of accounts grows, navigating your infrastructure and finding resources via the [Console](https://aws.amazon.com/console) or the [CLI](https://aws.amazon.com/cli/) becomes hard to impossible.
```

**Review by scapecast — APPROVED** [review]

File: blog/2022-02-04-resoto-query-language-101/index.md [hunk 2]
```diff
@@ -13,7 +12,7 @@ import Image from '@theme/IdealImage';
...
 
-Retrieving information about resources you have deployed in your [AWS](https://aws.amazon.com) infrastructure means tediously navigating the [AWS](https://aws.amazon.com) web console or using their CLI. Every product has its way of navigation and data to show. The process becomes cumbersome quickly, especially if you have infrastructure in more than one region or account since you need to switch explicitly for every combination.
+Retrieving information about resources you have deployed in your [AWS](https://aws.amazon.com) infrastructure means tediously navigating the [AWS Management Console](https://aws.amazon.com/console) or using the [AWS Command Line Interface](https://aws.amazon.com/cli). That approach works well in a single account setup. However, the recommended best practice for [AWS](https://aws.amazon.com) is to set up a multi-account environment - and as the number of accounts grows, navigating your infrastructure and finding resources via the [Console](https://aws.amazon.com/console) or the [CLI](https://aws.amazon.com/cli/) becomes hard to impossible.
```
Comment (TheCatLady) [inline blog/2022-02-04-resoto-query-language-101/index.md:15]:
```suggestion
Retrieving information about resources you have deployed in your [AWS](https://aws.amazon.com) infrastructure means tediously navigating the [AWS Management Console](https://aws.amazon.com/console) or using the [AWS Command Line Interface](https://aws.amazon.com/cli). This approach works well in a single account setup, but best practice is to set up a multi-account environment. As the number of accounts grows, navigating your infrastructure and finding resources via the [Console](https://aws.amazon.com/console) or the [CLI](https://aws.amazon.com/cli/) becomes hard to impossible.
```

**Pull request merged**